.github/workflows/tests.yml

```diff
@@ -48,7 +49,7 @@ jobs:
         run: yarn install --prefer-offline --frozen-lockfile --check-files
 
       - name: Procompile assets
-        run: bundle exec rails assets:precompile
+        run: bundle exec rails assets:precompile NODE_OPTIONS="--openssl-legacy-provider"
```

Comment:
Do we know which package is not compatible with a higher node yet?

Reply:
Think it's webpack, not 100% sure